cypress/integration/spec_configures_nodes.js

```diff
@@ -54,17 +54,17 @@ describe('Configuring nodes', () => {
         cy.get('#port').clear()
         cy.get('#port').type(conn["port"])
         cy.get('[value="test"]').click()
-        cy.get(':nth-child(2) > button > div').should('have.css', 'color', 'rgb(0, 128, 0)') // connectable: green
-        cy.get(':nth-child(5) > button > div').should('have.css', 'color', 'rgb(255, 0, 0)') // Credentials: red
+        // cy.get(':nth-child(2) > button > div').should('have.css', 'color', 'rgb(0, 128, 0)') // connectable: green
```

Comment:
Would be good to make a comment here, why you commented it out. I suppose because you will replace the css / imgs used here and / or add use data-cy attributes to test that.